apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryInsertResults.cs

```diff
@@ -135,14 +136,17 @@ public BigQueryInsertResults ThrowOnAnyError()
                 return this;
             }
 
-            var exception = new GoogleApiException(_client.Service.Name, $"Error inserting data. Status: {Status}")
+            var flattenedErrors = Errors.SelectMany(rowErrors => rowErrors).ToList();
+            throw new GoogleApiException(_client.Service.Name)
             {
                 Error = new RequestError
                 {
-                    Errors = Errors.SelectMany(rowErrors => rowErrors).ToList()
+                    Errors = flattenedErrors,
+                    Message = $"Error inserting data: {flattenedErrors.Count} error(s). " +
+                    $"Status: {Status}. " +
```

Comment:
Nit: indent these two lines (as they're part of Message)

Reply:
Done